dartagnan/src/main/java/com/dat3m/dartagnan/program/processing/Intrinsics.java

```diff
@@ -1328,6 +1334,18 @@ private List<Event> inlineMemCpy(FunctionCall call) {
         return replacement;
     }
 
+    private List<Event> inlineMemCpyC11(FunctionCall call) {
+        return memCpy(call, true);
+    }
+
+    private List<Event> inlineMemCpyC99(FunctionCall call) {
+        return memCpy(call, false);
+    }
+
+    private List<Event> inlineMemCpyLLVM(FunctionCall call) {
+        return memCpy(call, false);
+    }
+
```

Comment:
You could also use a lambda instead of these 3 version, i.e., rather than referencing the extra method `Intrinsics::inlineMemCpyLLVM` you could directly write `call -> memCpy(call, false)`.